core/node/eth_sender/src/eth_fees_oracle.rs

```diff
@@ -41,6 +42,16 @@ impl GasAdjusterFeesOracle {
         let priority_fee_per_gas = self.gas_adjuster.get_blob_tx_priority_fee();
         let blob_base_fee_per_gas = Some(self.gas_adjuster.get_blob_tx_blob_base_fee());
 
+        if base_fee_per_gas > self.max_acceptable_base_fee_in_wei {
+            tracing::info!(
+                    "base fee per gas: {} exceed max acceptable fee in configuration: {}, skip transaction",
+                    base_fee_per_gas,
+                    self.max_acceptable_base_fee_in_wei
+            );
+
+            return Err(ExceedMaxBaseFee);
+        }
+
```

Comment:
We should also cap it in later resending transaction `if let Some(previous_sent_tx) = previous_sent_tx {` check?

Reply:
for resent transaction, we shall ignore the cap otherwise they will never be able to get through according to current logic and be stuck indefintely

Reply:
if that is a case, maybe reduces the increment to align with  line 110 ?

Reply:
I don't get which increment you are talking about. This line is just to skip the transaction is the current network base fee is over max_acceprable_base_fee

Reply:
Oh okay, you are talking about the blob transactions

10% is the minimum transaction base fee increment to avoid replacement underprice error in case of non-blob transaction

For blob transaction, it may follow different logic. but the comment suggest we need to double the value to avoid error.

In any case I did not change the logic here because it does not matter for us, we are not going to publish blob on mainnet with our config